### ISSUE TITLE
aboat the label of ESR

### ISSUE THREAD

**Issue by Elijah-Yi** [yaojieliu/ECCV2020-STDN#16]
in train.py line 93,    The label of the living sample should be **1** and the label of the spoof should be **0**, in line 93**:** esr_loss = l1_loss(M_li, -1) + l1_loss(M_sp, 1), maybe: rewritten as esr_loss = esr_loss = l1_loss(M_li, 1) + l1_loss(M_sp, -1)

**Comment by Elijah-Yi** [comment]
Misunderstanding
